openvino_tensorflow/executable.cc

```diff
@@ -220,7 +220,7 @@ bool Executable::Call(const vector<shared_ptr<ov::Tensor>>& inputs,
       m_in_names[input_index] = input_name;
     }
   }
-  for (int i = 0; i < parameters.size(); i++) {
+  for (int i = 0; i < inputs.size(); i++) {
```

Comment:
This change makes ssd_resnet101_v1_fpn_640x640_coco17_tpu-8 fail

Reply:
Seems like it is passing on my side. Can you confirm if it is the TFHub model below?:
https://tfhub.dev/tensorflow/retinanet/resnet101_v1_fpn_640x640/1

Reply:
Right, my bad. I will push the fix soon.

Reply:
The fix is pushed.